IDSDeathBlossom.py

```diff
@@ -243,7 +243,13 @@
                     sys.exit(-3)
                 else:
                     IDSTool.targets.append(target)
-    
+
+    if options.list_engines:
```

Comment:
There is already code to do this: https://github.com/EmergingThreats/IDSDeathBlossom/blob/master/IDSDeathBlossom.py#L219